### ISSUE TITLE
Update FMU's

### ISSUE THREAD

**Issue by lvanfretti** [ALSETLab/Modelica-Drone-3D-FMI#17]
Hi @mpodlaski 
Now that we have it, we should use the source code generation license to  generate new FMUs.
I suggest you use Dymola 2020x, and maybe you can generate FMUs for the different cases, not only the "ideal power system" but also the other ones that require libraries.

**Comment by dietmarw** [comment]
@lvanfretti Is this still an open issue?

**Comment by lvanfretti** [comment]
@dietmarw yes, we should do this because all of the updates to the library. Let’s leave it open for now.
